classes/ActionScheduler_AdminView.php

```diff
@@ -58,12 +65,37 @@ public function init() {
 		add_filter( 'posts_search', array( self::instance(), 'search_post_password' ), 10, 2 );
 	}
 
-	public function action_scheduler_post_type_args( $args ) {
```

Comment:
As a public function, this needs to be deprecated, not simply deleted. It could be being used by 3rd party code, and if it's deleted, that will trigger a fatal error.

See examples in Subscriptions for how to deprecate a function, but in short:

1. move it to the end of the file
2. trigger a deprecated function notice